File: app/src/main/java/hypr/a255bits/com/hypr/Generator/Generator.java

package hypr.a255bits.com.hypr.Generator;

import com.squareup.moshi.Json;

public class Generator {

    @Json(name = "name")
    private String name;
    @Json(name = "file_size_in_bytes")
    private Integer fileSizeInBytes;
    @Json(name = "model_url")
    private String modelUrl;
    @Json(name = "generator")
    private Generator_ generator;
    @Json(name = "price_in_cents")
    private Integer priceInCents;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Integer getFileSizeInBytes() {
        return fileSizeInBytes;
    }

    public void setFileSizeInBytes(Integer fileSizeInBytes) {
        this.fileSizeInBytes = fileSizeInBytes;
    }

    public String getModelUrl() {
        return modelUrl;
    }

    public void setModelUrl(String modelUrl) {
        this.modelUrl = modelUrl;
    }

    public Generator_ getGenerator() {
        return generator;
    }

    public void setGenerator(Generator_ generator) {
        this.generator = generator;
    }

    public Integer getPriceInCents() {
        return priceInCents;
    }

    public void setPriceInCents(Integer priceInCents) {
        this.priceInCents = priceInCents;
    }

}

File: app/src/main/java/hypr/a255bits/com/hypr/Generator/Generator.kt
package hypr.a255bits.com.hypr.Generator

import android.os.Parcelable
import com.squareup.moshi.Json
import kotlinx.android.parcel.Parcelize

@Parcelize
class Generator : Parcelable {
    @Json(name = "name")
    var name: String = ""

    @Json(name = "file_size_in_bytes")
    var fileSizeInBytes: Long? = null

    @Json(name = "model_url")
    var model_url: String? = null

    @Json(name = "generator")
    var generator: Generator_? = null

    @Json(name = "price_in_cents")
    var priceInCents: Int? = null

    @Json(name = "google_play_id")
    var google_play_id: String = ""
}
